app/src/main/java/com/sunasterisk/anticovid_19/ui/statistic/StatisticsPresenter.kt

```diff
@@ -1,10 +1,14 @@
 package com.sunasterisk.anticovid_19.ui.statistic
 
+import android.content.Context
+import com.sunasterisk.anticovid_19.R
```

Comment:
Presenter không nên chứa Context hay bất kì thành phần nào của Android 
Truyền SharedPreferencesExtension vào data source giống như DAO, database

Reply:
Tks you